sdk/lib/opentelemetry/sdk/baggage/manager.rb

```diff
@@ -6,10 +6,10 @@
 
 module OpenTelemetry
   module SDK
-    module CorrelationContext
+    module Baggage
       # Manages correlation context
       class Manager
-        CORRELATION_CONTEXT_KEY = OpenTelemetry::CorrelationContext::Propagation::ContextKeys.correlation_context_key
+        CORRELATION_CONTEXT_KEY = OpenTelemetry::Baggage::Propagation::ContextKeys.baggage_key
```

Comment:
Similar to the above, is there rational I'm missing for keeping the naming correlation context here?